bundles/org.openhab.binding.netatmo/src/main/java/org/openhab/binding/netatmo/internal/handler/capability/HomeSecurityThingCapability.java

```diff
@@ -51,5 +54,15 @@ public void initialize() {
         super.initialize();
         securityCapability = handler.getHomeCapability(SecurityCapability.class);
         homeCapability = handler.getHomeCapability(HomeCapability.class);
+        ApiBridgeHandler accountHandler = handler.getAccountHandler();
+        if (accountHandler != null) {
+            webhook = accountHandler.getWebHookServlet();
+            webhook.ifPresent(servlet -> servlet.registerDataListener(handler.getId(), this));
+        }
+    }
+
+    @Override
+    public void dispose() {
+        webhook.ifPresent(servlet -> servlet.unregisterDataListener(handler.getId()));
```

Comment:
Perhaps `super.dispose()` should be called, even though it currently has no implementation? Otherwise, for consistency, `initialize()` probably shouldn't call `super.initialize()` either. So it depends on the intention of the `Capability` class.